### PR TITLE
Add coverages as referenced entities

### PR DESCRIPTION
**Pull request by pestaa**
* https://github.com/prezly/prezly/pull/15092

### CODE REVIEW

File: src/types/Story.ts [hunk 1]
```diff
@@ -282,6 +283,7 @@ export namespace Story {
 
         referenced_entities: {
             stories: Record<string, OEmbedInfo>;
+            coverages: Record<number, CoverageEntry>;
```
Comment (kudlajz) [inline src/types/Story.ts:286]: I believe `coverage` is correct as it's non-countable word: https://www.britannica.com/dictionary/coverage.
Also, did you know that word of the day is [goulash](https://www.britannica.com/dictionary/eb/word-of-the-day)?

**Review by kudlajz — APPROVED** [review]
Just make sure to fix the linter issue 🙂

File: src/types/Story.ts (hunk 1 above)
Comment (pestaa) [reply src/types/Story.ts:286]: Well, shoot, I noticed this late. Unfortunately, coverages is all over backend, so it made sense to follow the pattern. 

Similarly, I wouldn't hesitate to say goulashes.

Comment (kudlajz) [reply src/types/Story.ts:286]: What an absolute mad man.

**Comment by pestaa** [comment]
Thanks for pushing this.

**Pull request merged**